exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/join/MergeJoinBatch.java

```diff
@@ -127,17 +153,20 @@
     @Override
     public void update(int inputIndex) {
       switch(inputIndex) {
-        case 0:
-          leftSizer = new RecordBatchSizer(left);
-          leftRowWidth = leftSizer.netRowWidth();
+        case LEFT_INDEX:
+          setRecordBatchSizer(inputIndex, new RecordBatchSizer(left));
+          leftRowWidth = getRecordBatchSizer(inputIndex).netRowWidth();
+          logger.debug("mergejoin left incoming batch sizer : {}", getRecordBatchSizer(inputIndex));
```

Comment:
Maybe "Left incoming batch size"? (The log will tell you it is the merge join. The "sizer" is an implementation detail.)